server/src/main/resources/server-reference.conf

```diff
@@ -121,7 +121,7 @@ crossdata-server.akka.cluster.failure-detector.threshold = 12
 crossdata-server.akka.cluster.acceptable-heartbeat-pause = 20s
 crossdata-server.akka.cluster.heartbeat-interval = 6s
 
-
+crossdata-server.akka.http.server.parsing.max-content-length=infinite
```

Comment:
This is one of the problems with sending files in akka-http. We can set it up with the size we want.